.github/workflows/test.yml

```diff
@@ -76,13 +79,12 @@ jobs:
             - name: Grant MySQL permissions
               run: mysql -uroot -proot -e "grant all privileges on *.* to 'testing'@'%' with grant option;" --port 13306
 
-            - name: Install Version Specific Dependencies
-              if: ${{ ! matrix.experimental }}
-              run: composer require "laravel/framework:${{ matrix.laravel }}" --no-interaction --no-update
-
-            - name: Install Version Specific Dependencies (ignore-platform-req)
+            - name: Setup Composer args
               if: ${{ matrix.experimental }}
-              run: composer require "laravel/framework:${{ matrix.laravel }}" --no-interaction --no-update --ignore-platform-reqs
+              run: echo "COMPOSER_ARGS=--ignore-platform-reqs" >> $GITHUB_ENV
```

Comment:
This looks really weird, isnt there an easier way?

Reply:
Agreed, but nope, that's the official mechanism for doing it... https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable